### ISSUE TITLE
correct control sample

### ISSUE THREAD

**Issue by mortunco** [ReddyLab/CRADLE#95]
Hello,

Thank you for coming up with CRADLE. I would like to give it a try for our whole genome STARRseq run.

I was a little bit confused about the input of control bigwig file.

Given, I have a three types of samples, (a) DNA input lib, (b) STARRseq mrna control, (c) STARRseq mrna treatment. Using b and c for control and experiment would eliminate constitutively active enhancer regions. So I was wondering if its okay to make C/A and B/A to get "input" normalised counts.

Thanks in advance for your insight.
Best regards,

Tunc.

**Comment by mortunco** [comment]
Dear Young-Sook,

Thank you very much for your rapid response. In our previous [work](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02339-6) we found three main enhancer behavior. (treatment induced, constitutive active and inactive). Therefore, I would like to start as broad as possible and narrow down. To do that I thought if I give do a/b and a/c then i would have input normalized samples then I can do all further classification.

For example, prior to writing here, I used `MACS2 callpeaks` based on B vs A and C vs A to obtain peaks of B and C samples separately. If I did this with B vs C, then I would have gotten treatment induced enhancers only and miss the "steady state active" in your terms.

Based on your statement, could you help me understand how do you get type 1 (steady state ones ) if you run CRADLE without the input sample?  (ctrl B , exp C) (forgive my ignorance if its super obvious).

I appreciate your discussion. Its making me understand the whole thing super clear.

Best,

Tunc.

[hello](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02339-6)

**Comment by Young-Sook** [comment]
Hi Tunc,

I think we both might be confused at least partially due to using different terminology. I am sorry I didn't make it clear. What I meant by 'steady-state enhancer' is the enhancers that are active in 'B' relative to 'A', without considering 'C' relative to 'A'. If we do differential testing for 'C' vs 'B', then the resulting peaks are very likely to have the peaks active in 'B' relative to 'A'. 

I guess both  'constitutive active enhancers' and 'constitutive inactive enhancer' do not have differential activity (C vs B), but the difference of the two enhancers is whether they are active or inactive in B and C relative to A? Below is what I guess about 'constitutive active enhancers' and 'constitutive inactive enhancer'. Let me know if I am mistaken. 
```
constitutive active enhancers: the enhancers that are active in both B and C relative to A but do not have differential activity in C relative to B. 
constitutive inactive enhancer: the enhancers that are inactive (meaning not active and not repressed) in both B and C relative to A and do not have differential activity in C relative to B.
```

Based on my understanding, you want to have a set of peaks that are
1)  treatment-responsive enhancers (regardless of their read counts in DNA input lib—is it right?)
2) constitutive active enhancers
3) constitutive inactive enhancers

Once we are on the same page about your research goal, we can discuss how you can use CRADLE to detect the peaks you want.

If you want, we can do a quick zoom meeting too. Let me know.

Best,
Young-Sook

**Comment by mortunco** [comment]
Hi Young-Sook, 

**TL;DR**
I think one thing caused confusion which is the repression, lets forget about it for now because I think it would automatically solved if we figure out the increased activity case.

The following image would explain what I want to achieve. 
![image](https://user-images.githubusercontent.com/12833865/151714298-3ac6d934-f663-451e-b89a-4f768bdb22aa.png)

Exactly! Our previous optimization showed we have same amount of plasmids transfected to cells so DNA input would cancel each other if we calculate induction like this; (treatment/dna_input) / (control/dna_input).
```
treatment-responsive enhancers (regardless of their read counts in DNA input lib—is it right?)
constitutive active enhancers
constitutive inactive enhancers
```

**Detailed Responses to your comments:**
Your definition is correct. We are on the same page about this.
```
constitutive active enhancers: the enhancers that are active in both B and C relative to A but do not have differential activity in C relative to B. 
constitutive inactive enhancer: the enhancers that are inactive (meaning not active and not repressed) in both B and C relative to A and do not have differential activity in C relative to B.
```

I understand this. I think your confusion is caused by our definition of constitutive. We call an enhancer cons. if its only have steady-state enhancer activity in both conditions( control vs DNA, treatment vs DNA). Given this I agree with your statement below.
```
What I meant by 'steady-state enhancer' is the enhancers that are active in 'B' relative to 'A', without considering 'C' relative to 'A'. 
```

This is one point i am not sure. C vs B, could go both ways. if a enhancer higher at C then it has increased activity due to treatment which makes it induced. if its B > C, then its induced repression(we havent seen this before but we expect to see it in our whole genome library.
```
If we do differential testing for 'C' vs 'B', then the resulting peaks are very likely to have the peaks active in 'B' relative to 'A'.
```

**inactive,cons, induced explained in detail**
Inactive ones are inactive in both situations. The reason I wanna used DNA input is say if an input DNA count is 100 but we see 105 and 110 in control and treatment respectively, we would say there is no enhancer activity in this region as all plasmids entered the cell only produced 1 mRNA which is low.

Constitutively active ones, We see an increment with respect to DNA input but we dont see differential activity between control and treatment sample( which is exactly same with your definition in the quote box). For instance, DNA input is 100, control is 1000 and treatment is 1002.

Induced by treatment case as expected DNA input is 100, control can be 100 or 500 as long as treatment is 1000. There could be two cases, the enhancer is either only active in treatment which is DNA=100, control = 100 and treatment is 1000 or there is some basal enhancer activity in control but this has a drastic increase in the treatment such that DNA=100, control = 500, treatment 1000.


I am %100 down to meet on zoom. my 3mail is
tmorova "et" prostatecentre "daat" com (I apologise for next level encryption) 

Best,
Tunc.
